src/frontend/src/pages/AdminToolsPage/AdminToolsAttendeeDesignReviewModal.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import {
+  TextField,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  SelectChangeEvent,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow
+} from '@mui/material';
+import NERModal from '../../components/NERModal';
+
+interface AttendeeDesignReviewModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const AttendeeDesignReviewModal: React.FC<AttendeeDesignReviewModalProps> = ({ open, onClose }) => {
+  const [selectedTeam, setSelectedTeam] = useState('');
+  const [searchQuery, setSearchQuery] = useState('');
+  const tableHeaderCellStyle = {
+    borderRight: '2px solid white',
+    borderBottom: '2px solid white'
+  };
+
+  const tableBodyCellStyle = {
+    borderRight: '2px solid white'
+  };
+
+  const lastHeaderCellStyle = {
+    borderBottom: '2px solid white'
+  };
+
+  // TODO: to be deleted later, this is just stub data for filter options
+  const teams = ['All', 'Team A', 'Team B', 'Team C'];
+
+  // TODO: Stub data for team members, replace with dynamic data here
+  const teamMembers = [
+    { name: 'Batman', reviewsAttended: 2, missedReviews: 4 },
+    { name: 'Superman', reviewsAttended: 4, missedReviews: 1 }
+  ];
+
+  // TODO: Filtering team members based on search query here
+  const filteredMembers = teamMembers.filter((member) => member.name.toLowerCase().includes(searchQuery.toLowerCase()));
+
+  // TODO: Filtering for teams backend logic comes here
+  const handleTeamChange = (event: SelectChangeEvent) => {
+    setSelectedTeam(event.target.value as string);
+  };
+
+  const handleSearchChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchQuery(event.target.value);
+  };
+
+  return (
+    <NERModal
+      open={open}
+      onHide={onClose}
+      title="Attendee Design Review Information"
+      showCloseButton={true}
+      hideFormButtons={true}
+    >
+      <TextField
```

Comment:
this should have a formcontrol/ formlabel instead of using the label prop, same for the select below